include/unifex/thread_unsafe_event_loop.hpp

```diff
@@ -225,6 +235,16 @@ namespace _thread_unsafe_event_loop {
       return at_operation<std::remove_cvref_t<Receiver>>{
           (Receiver &&) r, dueTime_, loop_};
     }
+    template <typename Receiver>
+    at_operation<std::remove_cvref_t<Receiver>> connect(Receiver&& r) & {
+      return at_operation<std::remove_cvref_t<Receiver>>{
+          (Receiver &&) r, dueTime_, loop_};
+    }
+    template <typename Receiver>
+    at_operation<std::remove_cvref_t<Receiver>> connect(Receiver&& r) const& {
+      return at_operation<std::remove_cvref_t<Receiver>>{
+          (Receiver &&) r, dueTime_, loop_};
+    }
```

Comment:
Can we combine these overloads into a single `const&` overload to avoid the duplication?